src/state_private.h

```diff
@@ -13,6 +13,8 @@
 
 #define PROTOCOL_STR 16
 #define FUNC_MAX 24
+#define DLPI_MAX 256 
+#define REASON_MAX 256
```

Comment:
This needs to be the size of a max path + the size of the message. 
PATH_MAX + 128 
I think 128 is sufficient for existing messages

Reply:
done